src/Components/GEOSldas_GridComp/GEOSmetforce_GridComp/LDAS_Forcing.F90

```diff
@@ -4672,10 +4672,10 @@ subroutine GEOS_openfile(FileOpenedHash, fname_full, fid, tile_coord, m_hinterp,
        ! compute indices for nearest neighbor interpolation from GEOSgcm grid
        ! to tile space
           if( isCubed ) then ! cs grid
-             ! i_indg and j_indg are changed to LatLon grid
+             ! cube-sphere grid of forcing data must match cube-sphere grid associated with tile space
```

Comment:
I think we double-check this somewhere in ldas_setup or preprocessing.  Is that correct?

Reply:
No, it is not correct.  see my comment below

Reply:
After checking with @weiyuan-jiang , it turns out we do not verify that the cube-sphere grids in question match.   This is left for later.